devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java

```diff
@@ -310,19 +309,22 @@ public void execute() throws MojoFailureException, MojoExecutionException {
                     devModeContext.getModules().add(moduleInfo);
                 }
 
-                String resources = null;
-                for (Resource i : project.getBuild().getResources()) {
-                    //todo: support multiple resources dirs for config hot deployment
-                    resources = i.getDirectory();
-                    break;
-                }
+                /*
+                 * String resources = null;
+                 * for (Resource i : project.getBuild().getResources()) {
+                 * //todo: support multiple resources dirs for config hot deployment
+                 * resources = i.getDirectory();
+                 * break;
+                 * }
+                 */
```

Comment:
Hmmm. Either you remove this code or you add a comment explaining why it's commented but it really can't stay like that.

Reply:
There is a TODO comment which is why I didn't want to remove it. I can uncomment it if you prefer, it's a useless loop, atm.

Reply:
Ah sorry, I missed the TODO line.